src/components/Menu/Menu.tsx

```diff
@@ -101,6 +103,32 @@ export const MenuTrigger = forwardRef<
 ))
 MenuTrigger.toString = () => `.${MENU_TRIGGER_CLASS_NAME}`
 
+type MenuButtonProps = ComponentProps<typeof Button>
+const MENU_BUTTON_CLASS_NAME = 'c-menu-button'
+
+export const MenuButton = forwardRef<
+  ElementRef<typeof Trigger>,
+  MenuButtonProps
+>(({ children, ...props }, forwardedRef) => (
+  <Trigger className={MENU_BUTTON_CLASS_NAME} asChild ref={forwardedRef}>
+    <Button {...props}>
+      {children}
+      <ChevronDown
+        aria-hidden
+        css={{
+          position: 'relative',
+          top: 1,
+          '[data-state=open] &': { transform: 'rotate(-180deg)' },
+          '@media (prefers-reduced-motion: no-preference)': {
```

Comment:
this is good to include - I think  this should be made a utility function - but that doesn't need to happen in this PR - I'll record it.